client/my-sites/plan-features/header.jsx

```diff
@@ -43,7 +49,7 @@ class PlanFeaturesHeader extends Component {
 				</div>
 				<div className="plan-features__header-text">
 					<h4 className="plan-features__header-title">{ title }</h4>
-					<PlanFeaturesPrice currencyCode={ currencyCode } rawPrice={ rawPrice } />
+					{ currencyCode && <PlanFeaturesPrice currencyCode={ currencyCode } rawPrice={ rawPrice } /> }
```

Comment:
Fair enough 😄 It'd be more usable if PlanFeaturesPrice could handle a falsey value instead